contents/docs/self-host/index.mdx

```diff
@@ -8,7 +8,7 @@ showTitle: true
 import Disclaimer from './\_snippets/disclaimer.mdx'
 import Sunset from '../self-host/\_snippets/sunset-disclaimer.mdx'
 
-> **This page covers our open-source Docker compose deployment**, which is available under a MIT license, [without guarantee](/docs/self-host/open-source/disclaimer). We are continuing to develop this version of PostHog, but please note that PostHog does not provide support for customers deploying this version beyond these docs.
+> > **This covers our free, open-source Docker compose deployment**, which is available under a MIT license [without guarantee](/docs/self-host/open-source/disclaimer). We continue to develop this version, but do not provide support for it beyond these docs. We [no longer support paid, open-source deployments](/blog/sunsetting-helm-support-posthog) and it is **no longer possible to buy licenses for self-hosted versions** - we instead recommend [migrating to PostHog Cloud](/docs/migrate/migrate-to-cloud). 
```

Comment:
I'd say something along the lines of "premium features are only offered on PostHog cloud" so people know there are features missing.

Reply:
```suggestion
> **This covers our free, open-source Docker compose deployment**, which is available under a MIT license [without guarantee](/docs/self-host/open-source/disclaimer). We continue to develop this version, but do not provide support for it beyond these docs and some premium features are only available on PostHog Cloud. 
>
> We [no longer support paid, open-source deployments](/blog/sunsetting-helm-support-posthog) and it is **no longer possible to buy licenses for self-hosted versions** - we instead recommend [migrating to PostHog Cloud](/docs/migrate/migrate-to-cloud). 
```